app/src/main/java/net/sourceforge/opencamera/MyApplicationInterface.java

```diff
@@ -1906,6 +1906,7 @@ static float getPanoramaPicsPerScreen() {
     public void touchEvent(MotionEvent event) {
         main_activity.getMainUI().closeExposureUI();
         main_activity.getMainUI().closePopup();
```

Comment:
Is this line not enough to close popup on any touch event? Or maybe all the other click events in MainActivity are not recognized as this "touch event" and this is why you need to repeatedly write closePopup()?